about.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Breana Kambs Official Website</title>
+  <link rel="shortcut icon" type="image/png" href="images/favicon.png">
+</head>
+<body>
+  <hr /> 
+  <h1><img src="images/Kambs-Logo.png" alt="Kambs-Logo" width="67" height="38" align="left" align="middle" />
```

Comment:
I recommend adding a <header>, <main>, and <footer> element with the appropriate elements nested inside those tags. This will create more meaningful structure for your pages.